src/nvim/path.c

```diff
@@ -1761,7 +1761,7 @@ int pathcmp(const char *p, const char *q, int maxlen)
   /* ignore a trailing slash, but not "//" or ":/" */
   if (c2 == NUL
       && i > 0
-      && !after_pathsep((char_u *)s, (char_u *)s + i)
+      && !after_pathsep((char *)s, (char *)s + i)
```

Comment:
Same as before. Don't cast from const to non-const. Why not making `after_pathsep` arguments `const` too?